Include/ShaderConductor/ShaderConductor.hpp

```diff
@@ -161,13 +173,26 @@ namespace ShaderConductor
             const char* version;
         };
 
+		struct ReflectionDesc
+        {
+            char* name;					// Name of the resource
+            ShaderResourceType type;	// Type of resource (e.g. texture, cbuffer, etc.)
+            uint32_t bufferBindPoint;	// Buffer's starting bind point
+            uint32_t bindPoint;			// Starting bind point
+            uint32_t bindCount;			// Number of contiguous bind points (for arrays)
+        };
+
         struct ResultDesc
         {
             Blob* target;
             bool isText;
 
             Blob* errorWarningMsg;
             bool hasError;
+
+            ReflectionDesc* reflectionDescs;
```

Comment:
You can consider adding a new type, say, ReflectionResultDesc, to put these in just for reflection.